tools/install.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+
+# installation of the ProActive distribution
+ 
+read -e -p "Directory where to install the scheduler: " -i "/opt/proactive" PA_ROOT
+
+mkdir -p $PA_ROOT
+rm -f $PA_ROOT/default
+
+CURRENT_PADIR="$(dirname "$SCRIPT_DIR")"
+PA_FOLDER_NAME="$(basename "$CURRENT_PADIR")"
+
+/usr/bin/cp -f -R $CURRENT_PADIR $PA_ROOT
+
+ln -s -f $PA_ROOT/$PA_FOLDER_NAME $PA_ROOT/default
+
+# creation of the proactive user
+
+read -e -p "Name of the user starting the ProActive service: " -i "proactive" USER
+
```

Comment:
Shouldn’t we ask for default group associated to the user that is created? I think that by default it would be nice to use the same group and user as the one used for proactive-agent distribution .